physics/inertia_tensor_body.hpp

```diff
@@ -84,14 +90,48 @@ InertiaTensor<Frame>::Diagonalize() const {
   return {moments_of_inertia, swap_x_and_z * eigensystem.rotation};
 }
 
+template<typename Frame>
+InertiaTensor<Frame> InertiaTensor<Frame>::MakeWaterSphereInertiaTensor(
+    Mass const& mass) {
+  static constexpr Density ρ_of_water = 1000 * Kilogram / Pow<3>(Metre);
+  static constexpr MomentOfInertia zero;
+  MomentOfInertia const I =
+      Cbrt(9 * Pow<5>(mass) / (250 * Pow<2>(π * ρ_of_water)));
+  return InertiaTensor<Frame>(mass,
+                              R3x3Matrix<MomentOfInertia>({I, zero, zero},
+                                                          {zero, I, zero},
+                                                          {zero, zero, I}),
+                              Frame::origin);
```

Comment:
Wait, we are using `InertiaTensor<Barycentric>`, and putting these water spheres at the origin? Shouldn't they be at the origin of the part instead?

Alternatively, shouldn't we use `InertiaTensor<RigidPart>`, so that we don't store the position twice?

Reply:
Done (the latter).